internal/ast/types.go

```diff
@@ -666,6 +667,33 @@ type EnumType struct {
 	Values []EnumValue // possible values. Value types might be different
 }
 
+func (t EnumType) MemberForValue(value any) (EnumValue, bool) {
+	if len(t.Values) == 0 {
+		return EnumValue{}, false
+	}
+
+	if value == nil {
+		return t.Values[0], false
```

Comment:
Its assuming that its the default one?

Reply:
Kinda but not really, it's just to return "something" that can be used as a valid empty value for an enum.